General_gNMI/GetLibrary.py

```diff
@@ -163,6 +163,8 @@ def check_last_updates_not_include(self, text: str) -> bool:
 
     def _last_updates_not_empty(self) -> bool:
         last_updates = self.get_last_flattened_updates()
+        if not last_updates:
+            return False
```

Comment:
Do you think this is needed? The subsequent `any()` terminates immediately if there are no updates.

Reply:
was buggy, thanks - fixed the method now to behave according to original intent...